lib/components/Masthead/Masthead.tsx

```diff
@@ -76,20 +76,21 @@ export interface MastheadProperties {
 // Root container
 const MastheadContainer = styled(Attr.div)`
     &&&& {
-        color: ${props => props.theme.colorHeaderTextRest};
-        background-color: ${props => props.theme.colorHeaderBackground };
+        color: ${(props: ThemeProps<ShellTheme>) => props.theme.colorTextMastheadRest};
+        background-color: ${(props: ThemeProps<ShellTheme>) => props.theme.colorBgMasthead };
     }
 `;
 
 // Translates Shell theme to action button theme under current context.
-function toolbarButtonTheme(theme: ShellTheme): ActionTriggerButtonTheme {
+function toolbarButtonTheme(theme: ShellTheme): ShellTheme {
     return theme ? {
-        colorRest: theme.colorHeaderButtonRest,
-        colorHover: theme.colorHeaderButtonHover,
-        colorDisabled: theme.colorHeaderTextRest,
-        colorTextRest: theme.colorHeaderTextRest,
-        colorTextDisabled: theme.colorHeaderTextDisabled
-    } : {}; // Theme must be an object.
+        base: theme.base,
+        colorBgBtnStandardRest: theme.colorBgMasthead,

```

Comment:
nit: is this suppose to map colorBgMasthead to colorBgBtnStandardRest

Reply:
correct, this is one special case, because under masthead context, the action trigger button will show same color as masthead, so we need this theme translation for the buttons.